src/main/java/net/neoforged/neoforge/registries/GameData.java

```diff
@@ -65,7 +65,10 @@ public static void unfreezeData() {
 
     public static void freezeData() {
         LOGGER.debug(REGISTRIES, "Freezing registries");
-        BuiltInRegistries.REGISTRY.stream().filter(r -> r instanceof MappedRegistry).forEach(r -> ((MappedRegistry<?>) r).freeze());
+        BuiltInRegistries.REGISTRY.stream().filter(r -> r instanceof MappedRegistry).forEach(r -> {
+            ((MappedRegistry<?>) r).bindAllTagsToEmpty();
```

Comment:
Leave a comment here detailing why we call `bindAllTagsToEmpty` since it's non-obvious from the context.